rfd/0012-teleport-versioning.md

```diff
@@ -141,3 +141,32 @@ story. We never want a `vN+1` that has no migration path other than "rebuild
 the cluster from scratch".
 
 So we don't want to freeze the major version forever this way.
+
+### Docker image tags
+
+Currently, we have 2 kinds of docker images:
+
+- `teleport:X.Y.Z` - immutable tag for a specific patch release (for example
+  `4.4.1`, `4.4.2`)
+- `teleport:X.Y` - mutable tag pointing at the latest patch release for `X.Y` (for
+  example `4.4` pointing at `4.4.2`)
+
+For new versioning, the following tags are created:
+
+- `teleport:X.Y.Z` - immutable tag for a specific patch release (for example
+  `5.1.0`, `5.1.1`)
+- `teleport:X.Y` - mutable tag pointing at the latest patch release for minor
+  version `X.Y` (for example `5.1` pointing at `5.1.1`)
```

Comment:
To be clear, I think we should just stop offering these to encourage people to switch to the new `teleport:X` format.